sample/client/dev_token.py

```diff
@@ -53,9 +64,27 @@
 for notebook in notebooks:
     print "  * ", notebook.name
 
-print
-print "Creating a new note in the default notebook"
-print
+#list all the notes in the default notebook:
+default_notebook = note_store.getDefaultNotebook()
```

Comment:
You can do this without adding an additional API call - When iterating through eh list of `notebooks` above, check the `defaultNotebook` for each; the default notebook will have it set to `true`.